Automattic/wp-calypso
# Domain Transfer: Use inline instructions for tooltip modal on domains step

## Proposed Changes

* Now that we have the modal for Google Domains transfer instructions, we should use this where appropriate, such as on the auth code information tooltip.

## Testing Instructions

* Go to `/setup/domain-transfer/domains`
* Click auth code tooltip and verify learn more link goes to a support doc
* Go to `setup/google-transfer/domains` 
* Click auth code tooltip and verify the the transfer instructions show with the auth code panel opened by default.

**Google**

<img width="1159" alt="Screenshot 2023-07-30 at 4 38 59 PM" src="https://github.com/Automattic/wp-calypso/assets/1126811/13cfd531-f297-4390-8081-13d47f5e416e">

**All others**

<img width="1158" alt="Screenshot 2023-07-30 at 4 39 22 PM" src="https://github.com/Automattic/wp-calypso/assets/1126811/9b3bef06-7a23-487a-a9b7-08c09ce4f13a">



## Pre-merge Checklist
